CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ### Notable changes
 
+### Breaking changes
+
+* DCOS_METRICS_-prefixed labels are now configured to be included in metrics (DCOS-43591)
```

Comment:
Is this a breaking change? What would it break?

Reply:
Evan brought this up in the telegraf PR that this might be considered a breaking change so I included it under this section - since these labels were previously included by default, but now it's configurable (so technically not included by default anymore). Practically there is no difference here since we updated the dcos configs to include `DCOS_METRICS_` prefix.

Reply:
removed